db/migrate/20250314102616_rename_allow_anonymous_posting_to_allow_anonymous_mode.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RenameAllowAnonymousPostingToAllowAnonymousMode < ActiveRecord::Migration[7.2]
+  def up
+    execute(<<~SQL)
+      WITH renamed AS (
+        DELETE FROM site_settings
+        WHERE name = 'allow_anonymous_posting'
+        RETURNING 'allow_anonymous_mode' AS name, data_type, value, created_at, updated_at
+      )
+      INSERT INTO site_settings
+      (name, data_type, value, created_at, updated_at)
+      SELECT * FROM renamed
```

Comment:
Can this be simple rename? Something like
```
execute "UPDATE site_settings SET name = 'allow_anonymous_mode' where name = 'allow_anonymous_posting'"
```

Reply:
🤦‍♂️ oh my god it's so simple what is this overengineered mess that I've done 😭